src/org/omegat/Main.java

```diff
@@ -122,6 +128,10 @@ public static void main(String[] args) {
             TeamTool.main(Arrays.copyOfRange(args, 1, args.length));
         }
 
+        // retrieve JVM parameters.
+        RuntimeMXBean runtimeMxBean = ManagementFactory.getRuntimeMXBean();
+        JVMPARAMS.addAll(runtimeMxBean.getInputArguments());
```

Comment:
Is there a reason we need to store these at startup? It seems that it's probably only used at restart time, so we might as well just wait until then.

Reply:
Yes, it is only used at restart time.